### PR TITLE
adding resources in free-courses-en.md

### PR DESCRIPTION
**Pull request by PranavKrishnan007**
## What does this PR do?
Add resource(s) in free-courses-en.md

### Why is this valuable (or not)?
Yes! :D

### How do we know it's really free?
cause it is freely available on youtube

### For book lists, is it a book? For course lists, is it a course? etc.
these are detailed courses .
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -675,6 +676,7 @@
 * [Data Processing and Feature Engineering with MATLAB](https://www.coursera.org/learn/feature-engineering-matlab) (coursera)
 * [Introduction to Matlab in English](https://www.youtube.com/playlist?list=PLGED90Y_uL1KLpdRmVtwfpNoYCWU9RPkK) -Mohammed Mohammed
 * [MATLAB and Simulink Training](https://matlabacademy.mathworks.com)
+* [MATLAB Programming for Numerical Computation NPTEL](https://www.youtube.com/playlist?list=PLRWKj4sFG7-6_Xr9yqg6SMr_F80KdFVhN) - MATLAB Programming for Numerical Programming (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:679]: "MATLAB Programming for Numerical Programming" is ot an author

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -250,6 +250,7 @@
 
 ### Compilers
 
+* [Compilers](https://www.youtube.com/playlist?list=PL6KMWPQP_DM97Hh0PYNgJord-sANFTI3i) - Ghassan Shobaki Computer Science Lectures (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:253]: "Computer Science Lectures "is not an author

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (PranavKrishnan007) [reply courses/free-courses-en.md]: hope the last commit solves this issue.

File: courses/free-courses-en.md (hunk 2 above)
Comment (PranavKrishnan007) [reply courses/free-courses-en.md]: i've changed the name of the author

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -675,6 +676,7 @@
 * [Data Processing and Feature Engineering with MATLAB](https://www.coursera.org/learn/feature-engineering-matlab) (coursera)
 * [Introduction to Matlab in English](https://www.youtube.com/playlist?list=PLGED90Y_uL1KLpdRmVtwfpNoYCWU9RPkK) -Mohammed Mohammed
 * [MATLAB and Simulink Training](https://matlabacademy.mathworks.com)
+* [MATLAB Programming for Numerical Computation NPTEL](https://www.youtube.com/playlist?list=PLRWKj4sFG7-6_Xr9yqg6SMr_F80KdFVhN) - Dr. Niket Kaisare NPTEL (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:679]: Authors name is not "Dr. Niket Kaisare NPTEL" . This repo mostly doesn't use honorifics like 'Dr.' or 'Prof.".

Comment (PranavKrishnan007) [reply courses/free-courses-en.md:679]: could you please suggest the author name .... like would "Niket Kaisare NPTEL" do?

**Comment by eshellman** [comment]
please remove honorifics.

**Comment by PranavKrishnan007** [comment]
I have removed the honourifics ....hoping these changes are sufficient

**Comment by eshellman** [comment]
Thanks!

**Pull request merged**